### ISSUE TITLE
Inconsistent documentation for `cargo install-update-config`

### ISSUE THREAD

**Issue by TehPers** [nabijaczleweli/cargo-update#197]
`cargo install-update-config --help` returns documentation containing the following usage string:

```
USAGE:
    cargo install-update-config [FLAGS] [OPTIONS] <PACKAGE>
```

However, if you try to do `cargo install-update-config --feature extra nu`, you get the following error:

```
error: The following required arguments were not provided:
    <PACKAGE>

USAGE:
    cargo install-update-config <PACKAGE> --feature <FEATURE>...

For more information try --help
```

The help text and the error message put the options in different locations. It would be nice if the options could be specified either before or after the package name, but for now it might be helpful to correct the help string so that the `<PACKAGE>` appears before `[FLAGS]` and `[OPTIONS]`.

I am using **v9.0.0** installed through `cargo install cargo-update`.

**Comment by TehPers** [comment]
Tested with a9aba93 and it seems to be working. Thanks!

**Comment by nabijaczleweli** [comment]
Fixed in [`v10.0.0`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v10.0.0)
